src/WebJobs.Script/Workers/Http/Configuration/HttpWorkerOptions.cs

```diff
@@ -15,8 +15,16 @@ public class HttpWorkerOptions
 
         public int Port { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the host will rebuild the initial invocation HTTP Request and send the copy to the worker process.
+        /// </summary>
         public bool EnableForwardingHttpRequest { get; set; }
```

Comment:
A bit unclear about the property name and the summary comment. From the name, it sounds like the request is simply being forwarded as it is. But the comment suggests that a new copy of the request is being rebuilt from the original and that is sent to the worker process. Should we consider updating the property name to reflect that behavior?

Reply:
I think you can update the comment to 

```
/// <summary>
/// Gets or sets a value indicating whether the host will forward the request to the worker process. 
/// </summary>
/// <remarks>
/// The host will rebuild the initial invocation HTTP Request and send the copy to the worker process.
/// </remarks>
```